test/unit/gateways/merchant_e_solutions_test.rb

```diff
@@ -16,13 +16,17 @@ def setup
 
     @options = {
       order_id: '1',
+      recurring_pmt_num: 11,
+      recurring_pmt_count: 10,
       billing_address: address,
       description: 'Store Purchase'
     }
   end
 
   def test_successful_purchase
     @gateway.expects(:ssl_post).returns(successful_purchase_response)
+    assert_equal @options[:recurring_pmt_num], 11
+    assert_equal @options[:recurring_pmt_count], 10
```

Comment:
This is just going to assert the values are equal to what they are set to in the setup. I was thinking something like this [test](https://github.com/activemerchant/active_merchant/blob/master/test/unit/gateways/adyen_test.rb#L174-L176). In that test, we are checking that the data is what we expect in the out-bound request.

Reply:
@naashton the test you just highlighted they are checking the values of response but for our case I have checked the responses we are not getting these keys from responses.